File: mycollab-web/src/main/scala/com/mycollab/module/project/events/CalendarEvent.scala
package com.mycollab.module.project.events

import com.mycollab.events.ApplicationEvent
import com.mycollab.module.project.domain.criteria.ProjectAssignmentSearchCriteria

/**
  * @author MyCollab Ltd
  * @since 5.2.1
  */
object CalendarEvent {
  
  class SearchRequest(source: AnyRef, data: ProjectAssignmentSearchCriteria) extends ApplicationEvent(source, data) {};
}

File: mycollab-web/src/main/java/com/mycollab/module/project/event/CalendarEvent.kt
package com.mycollab.module.project.event

import com.mycollab.events.ApplicationEvent
import com.mycollab.module.project.domain.criteria.ProjectTicketSearchCriteria

/**
 * @author MyCollab Ltd
 * @since 6.0.0
 */
class CalendarEvent {
    class SearchRequest(source: Any, data: ProjectTicketSearchCriteria) : ApplicationEvent(source, data)
}